### PR TITLE
Minor bug fix in iFBTSVM._decrement() method

### PR DESCRIPTION
**Pull request by wjdgns0914**
Please refer to #42 for the details

Env info
Windows-10-10.0.19045-SP0
Python 3.8.16 (default, Jun 12 2023, 21:00:42) [MSC v.1916 64 bit (AMD64)]
NumPy 1.24.1
SciPy 1.9.2
Scikit-Learn 1.2.0

### CODE REVIEW

**Comment by kritchie** [comment]
Could you please make sure the checks are passing ? 
I believe there's some unit tests that would need to be updated as well. :pray:

**Comment by wjdgns0914** [comment]
I found there are test codes in the library and the error is from test_update_model.py.
Typically, we would need to modify the code for passing the test, but I believe we need to modify the test in this case.

Because the key point of the issue I wrote is that we need to index `alpha`, `data`, and `fuzzy` through `score[0] ` not directly from `candidates`. Therefore, `score[0]` needs to be updated based on changes in the shape of `alpha`, `data`, and `fuzzy`. So, the ground truth of `score` in `test_decrement()` is `np.asarray([[0, 1], [2, 3]])` not `np.asarray([[1, 2], [2, 3]])`

Please check the changes in the test code, Thanks!

File: libifbtsvm/libifbtsvm.py [hunk 1]
```diff
@@ -65,19 +65,16 @@ def _compute_score(cls, score, c):
...
-        """
+        candidates_for_alphas = score[0][candidates].astype(int)
+        alphas = np.delete(alphas, candidates_for_alphas)
+        fuzzy = np.delete(fuzzy, candidates_for_alphas)
+        data = np.delete(data, candidates_for_alphas, axis=0)
 
-        :return:
-        """
         sco0 = np.delete(score[0], candidates)
+        for i in range(len(sco0)):  # update index arrary about alphas, after deletion on alphas
```
Comment (kritchie) [inline libifbtsvm/libifbtsvm.py:74]: This introduction of a non-vectorized `numpy` function could incur some non-negligible slowdown of the code. 

I haven't dig deep in the `numpy` library's documentation but perhaps there exist some function that would do the trick here. 

I'll let you judge if you think it's outside the scope of the current PR. However, to a minimum, I would request a `# TODO` comment to make sure this is updated eventually.

Comment (wjdgns0914) [reply libifbtsvm/libifbtsvm.py]: That's a very good point! I didn't think that far. Based on your suggestion, I think we can do vectorize like the new commit, what do you think?

File: libifbtsvm/libifbtsvm.py [hunk 2]
```diff
@@ -445,3 +444,4 @@ def score(self, X, y, sample_weight=None):
         :return: Accuracy score of the classification
         """
         return accuracy_score(y, self.predict(X), sample_weight=sample_weight)
+
```
Comment (wjdgns0914) [inline libifbtsvm/libifbtsvm.py:447]: Nothing changed here (just a blank line)

File: libifbtsvm/libifbtsvm.py (hunk 1 above)
Comment (kritchie) [reply libifbtsvm/libifbtsvm.py]: Sorry for the delay again (life :sweat_smile:)

The change looks fine to me. :+1:

**Comment by kritchie** [comment]
This PR looks good to me :+1: 
Thank you very much for this, it's really appreciated !

**Pull request merged**